manifests/gcp_marketplace/schema.yaml

```diff
@@ -108,6 +108,51 @@ properties:
     type: string
     x-google-marketplace:
       type: NAMESPACE
+  managedstorage.enabled:
+    type: boolean
+    title: Use managed storage
+    description: |-
+      Use Cloud SQL and GCS for storing the data.
+      Using CloudSQL and GCS provides better reliability and performance,
+      as well as features such as data backup and usage monitoring.
+      This is the recommended option especially for production scenarios.
+      If false, the data will be stored in GCE Persistent Disk.
+    default: false
+  managedstorage.cloudsqlInstanceConnectionName:
+    type: string
+    title: Cloud SQL instance connection name (Managed storage only)
+    description: |-
+      This field must be specified if choose to use managed storage.
+      Provide the instance connection name for an existing Cloud SQL for MySQL instance.
+      The instance connection name can be found on the instance detail page in the Cloud SQL console.
+      The instance connection name uses the format project:zone:instance-name, for example,myproject:us-central1:myinstance.
+      For more details on how to create a new instance, see https://cloud.google.com/sql/docs/mysql/quickstart.
+  managedstorage.dbUsername:
+    type: string
+    title: Database username (Managed storage only)
+    description: |-
+      The database username to use when connecting to the Cloud SQL instance.
+      If you leave this field empty, the deployment will use the default 'root' user account to connect.
+      For more details about MySQL users, see https://cloud.google.com/sql/docs/mysql/users.
+  managedstorage.dbPassword:
+    type: string
+    title: Database password (Managed storage only)
+    x-google-marketplace:
+      type: MASKED_FIELD	
+    description: |-
+      The database password to use when connecting to the Cloud SQL instance.
+      If you leave this field empty, the deployment will try to connect to the instance without providing a password.
+      This will fail if a password is required for the username you provided.
+  managedstorage.databaseNamePrefix:
```

Comment:
@rui5i In the future you might want to pass the DB name to the cache-server when it's deployed in the marketplace.

Reply:
I will update this PR to include guide on how to test locally before create a PR. Hope it helps.
Currently we only put test to post-submit to save pre-submit resources. (it can be in pre-submit)

Reply:
suggest adding it as a presubmit test that doesn't always run

Reply:
ack